portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,32 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function addAndReplaceRandomFact() {
```

Comment:
nit: addOrReplaceRandomFact? (I think you're either adding a fact if this is the first press or replacing it on subsequent presses).

Same for addPhoto.